source/fundamentals/logging.txt

```diff
@@ -351,7 +331,7 @@ a third-party logging package, into your application.
       specifications:
       
       - The logger writes log messages to the console.
-      - The logger logs messages at ``DEBUG`` level.
+      - The logger writes messages at the ``DebugLevel`` level.
```

Comment:
Suggestion:
I think it could be good to stay consistent in describing the action the logger takes ("emits" / "logs" / "writes"). I would probably choose "logs" since it more precisely describes what a "logger" accomplishes.